Autopilot/Inc/SensorFusion.hpp

```diff
@@ -0,0 +1,62 @@
+/**
+ * Sensor Fusion - Converts raw sensor data into human readable formatted structs
+ * Author: Lucy Gong, Dhruv Rawat, Anthony Bertnyk
+ */
+
+#include "AttitudeDatatypes.hpp"
+#include "fetchSensorMeasurementsMode.hpp"
+
+#ifndef SENSORFUSION_HPP
+#define SENSORFUSION_HPP
+
+// -1 = FAILED
+// 0 = SUCCESS
+// 1 = Old Data
+struct SFError_t{
+    int errorCode;
+};
+
+struct SFOutput_t {
+    float IMUroll, IMUpitch, IMUyaw; //rad
+    float IMUrollrate, IMUpitchrate, IMUyawrate; //rad/s
+    float Airspeed; //m/s
+    float altitude; //m
+    float altSpeed; //m/s
+    long double latitude; //Decimal degrees
+    float latSpeed; //m/s
+    long double longitude; //Decimal degrees
+    float longSpeed; //m/s
+};
```

Comment:
Should also have heading in here :))

Reply:
heading == yaw as long as it's measured as an azimuth